addons/necropost-highlighter/userscript.js

```diff
@@ -29,13 +29,11 @@ export default async function ({ addon, global, console, msg }) {
     applyToOpenSourceProjects: "Open Source Projects",
   };
   const searchResultsPageName = "Search Results"; // recent posts and unanswered posts
-  // mobile urls look like: https://scratch.mit.edu/discuss/m/6/*
-  const isMobileSite = /^\/discuss\/m\//.test(location.pathname);
-  // tableBodyNode will be null on mobile and non-table search page:
+  // tableBodyNode will be null on non-table search page:
   // https://scratch.mit.edu/discuss/search/?action=show_user&show_as=posts
-  const tableBodyNode = document.querySelector("tbody"); // null on mobile
+  const tableBodyNode = document.querySelector("tbody");
 
-  if (!tableBodyNode && !isMobileSite) {
+  if (!tableBodyNode) {
     // Standard site page without a table. Don't run on this page.
```

Comment:
This comment could be simplified as we don't have to account for the mobile forums anymore:
```suggestion
    // No table on this page. Don't run.
```